resources/management/commands/generate_winter_places.py

```diff
@@ -14,27 +14,42 @@
 )
 
 WINTER_AREAS_MAP = {
-    # TODO: update temporary values here
-    # from_place_number, to_place_number are just in basic order,
-    # modify these numbers when more we have more specs
+    # The data has the following shape:
+    # (section_identifier, (from_place_number, to_place_number), width, length)
     "Iso-Sarvasto": [
-        ((1, 29), 2.5, 6.0),  # ((from_place_number, to_place_number), width, length)
-        ((30, 69), 3.0, 8.0),
-        ((70, 85), 3.5, 10.0),
-        ((86, 89), 4.5, 12.0),
+        ("A", (1, 14), 2.5, 6.0,),
+        ("B", (15, 22), 3.0, 8.0),
+        ("B", (23, 23), 2.5, 6.0,),
+        ("C", (24, 37), 2.5, 6.0,),
+        ("ALAKENTTÄ", (1, 19), 3.0, 8.0),
```

Comment:
nitpick, but I would keep the identifier in pascal case, i.e. `Alakenttä`, we already have them like that for piers (e.g. `Rantamuuri`)

Reply:
Right! Didn't notice that 👍